src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -104,8 +104,8 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		@Inject(ICapabilitiesService) private capabilitiesService: ICapabilitiesService,
 		@Inject(ITextFileService) private textFileService: ITextFileService,
 		@Inject(ILogService) private readonly logService: ILogService,
-		@Inject(ITelemetryService) private telemetryService: ITelemetryService,
-		@Inject(ICommandService) private commandService: ICommandService
+		@Inject(ICommandService) private commandService: ICommandService,
+		@Inject(IAdsTelemetryService) private adstelemetryService: IAdsTelemetryService
```

Comment:
Make sure you go through all the tests that were creating NotebookModels and update them to use the NullAdsTelemetryService

Reply:
Done